# Don't use the xlocale.h header since it is an OS package and not always provided.
# R. van Dijk, 2018-03-07
--- src/src/data.c.orig	2018-03-06 05:49:34.973524976 -0500
+++ src/src/data.c	2018-03-07 08:03:08.097947515 -0500
@@ -53,9 +53,6 @@
 
 #include <string.h>
 #include <wchar.h>
-#ifndef WIN32
-#include <xlocale.h>
-#endif
 #include "migration.h"
 #include "sighandler.h"
 #include "tools.h"
